packages/tabs/src/elements/TabPanel.tsx

```diff
@@ -11,6 +11,9 @@ import { StyledTabPanel } from '../styled';
 import { useTabsContext } from '../utils/useTabsContext';
 
 interface ITabPanelProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * A value to uniquely identity a TabPanel. Provided to the `onChange` event.
```

Comment:
This lacks clarity. The `item` is what connects the tab panel to its associated tab.

Reply:
I've updated this wording.